content/it/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -1,157 +1,92 @@
 ---
-title: Che cos'è Kubernetes? 
+reviewers:
+- bgrant0607
+- mikedanese
```

Comment:
One tiny nit: can you remove this block?

Reply:
Hi @rlenferink  done 
thanks for the support